lib/web_ui/lib/src/engine/semantics/link.dart

```diff
@@ -18,9 +18,8 @@ class Link extends PrimaryRoleManager {
   @override
   DomElement createElement() {
     final DomElement element = domDocument.createElement('a');
-    // TODO(chunhtai): Fill in the real link once the framework sends entire uri.
-    // https://github.com/flutter/flutter/issues/102535.
-    element.setAttribute('href', '#');
```

Comment:
what will happen if there isn't a href?

Reply:
When you click on it, the browser doesn't navigate. It's still announced as a link by VoiceOver.